### PR TITLE
Nerfed E-Shield

### PR DESCRIPTION
**Pull request by Radezolid**
## About the PR
Modified values on the e-shield to make it thougher in health (3x) and resistances, but it reflects only 50% of the time rather than 95%.

## Why / Balance
Why? Because Lyndo was too lazy to PR it so I did it for them.

Balance:
With the upcoming devastator RPG replacement + most of the crew weapons being lasers it's kinda a mindless buy to nullify those weapons.

## Technical details
YAML changes.

## Media

https://github.com/user-attachments/assets/400254dd-84d9-4b4f-957c-8c31394c0497

## Requirements

- [x] I have tested all added content and changes.
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

None.

**Changelog**

:cl:
- tweak: The E-Shield now has a 50% reflect chance but it's durability is 3 times bigger.

### CODE REVIEW

**Comment by Lyndomen** [comment]
And resistances went up. Also it wasn't that I'm lazy, I'm just in bed and webedits are now illegal

**Comment by Radezolid** [comment]
> And resistances went up. Also it wasn't that I'm lazy, I'm just in bed and webedits are now illegal

You lazy :3

**Comment by CybersunOperative** [comment]
Hi. Hello. Direction here. This has probably been a long time coming nerf for the energy shield. The trade off for the reduction in reflect being additional shield HP seems fine enough. A competent operator will get more out of their shield than someone just mindlessly tanking bullets now, which is ideal.

Direction approves :3

**Comment by Avalon-Proto** [comment]
I still feel 50 is too big of a nerf- for a 10tc items that costs more than the weapon you pair with it in most cases, I feel a reduction to the 65-75 area is more reasonable. 95 was ridiculous yes, but cutting it down to nearly half of what it once was makes this item too expensive to be worth it

File: Resources/Prototypes/Entities/Objects/Shields/shields.yml [hunk 1]
```diff
@@ -418,16 +418,16 @@
       color: blue
     - type: ItemTogglePointLight
     - type: Reflect
-      reflectProb: 0.95
+      reflectProb: 0.5
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/Objects/Shields/shields.yml]: all of these need deltav comments

**Review by deltanedas — CHANGES_REQUESTED** [review]

**Comment by Radezolid** [comment]
Half asleep comments lmao

**Review by deltanedas — APPROVED** [review]
inb4 price reduction pr now that its a meme

**Comment by jink23** [comment]
> Hi. Hello. Direction here. This has probably been a long time coming nerf for the energy shield. The trade off for the reduction in reflect being additional shield HP seems fine enough. A competent operator will get more out of their shield **than someone just mindlessly tanking bullets** now, which is ideal.
> 
> Direction approves :3

You mean, Doing what a shield is supposed to do, block damage. If most the station is using lasers and the item costs 10tc it should be 'mindlessly blocking bullets' because that's what a shield SHOULD be doing having it be more durable isn't really much of a tradeoff

**Pull request merged**